docs/index.md

```diff
@@ -1,3 +1,43 @@
-## The UW MRO automated focusing system
+# The UW MRO automated focusing system
 
-These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG).
+These pages will describe the automated focusing system being developed by students in UW Astronomy Undergrad Engineering Group (AUEG). We are writing in python. There are two ways to achieve focusing:  
+a) measure FWHM and take the smallest value  
+b) look for the brightest pixels in the image to calculate radius of star and go from there
```

Comment:
The "brightest pixel" method does not require that you "calculate radius of star". It's based entirely on getting as much light focused onto a single pixel as possible. If you're going to mention it here, it would be good to also mention it below as something to check in the focus images, as a comparison with the star FWHM.